assets/src/application/ui/input/InlineEditInput/style.scss

```diff
@@ -25,8 +17,27 @@
 			height: 2.625rem; // 42px
 			width: 2.625rem; // 42px
 
+			.ee-rspnsv-table & {
```

Comment:
I decided to collocate this here, because when I made some changed to infinity icon, I wasn't aware that those changes might impact other areas;
having focus on single component allow to make this kind of approach, because the stylesheet for this component will be small, thus maintainable